### ISSUE TITLE
Doubly LinkedList class Implementation in CPP with all basic functions

### ISSUE THREAD

**Issue by Rohanfizz** [DankIndianCoders/Algorithms#2]
Please assign this issue to me :)

**Comment by TishaJhabak1014** [comment]
@thesmartdeveloperr Let me know if I can create a issue for contributing code for divide and conquer algo. Thank you.

**Comment by TishaJhabak1014** [comment]
> @thesmartdeveloperr Let me know if I can create a issue for contributing code for divide and conquer algo. Thank you.

Can do this contribution? : )

**Comment by thesmartdeveloperr** [comment]
> > @thesmartdeveloperr Let me know if I can create a issue for contributing code for divide and conquer algo. Thank you.
> 
> Can do this contribution? : )

sure @TishaJhabak1014 go ahead. which divide and conquer algo you are adding?

**Comment by thesmartdeveloperr** [comment]
@Rohanfizz , pls try to add only algorithms(and not any DS implementations), as the name for this repo suggests. will be reviewing and merging this one though

**Comment by TishaJhabak1014** [comment]
> > > @thesmartdeveloperr Let me know if I can create a issue for contributing code for divide and conquer algo. Thank you.
> > 
> > Can do this contribution? : )
> 
> sure @TishaJhabak1014 go ahead. which divide and conquer algo you are adding?

How about merge sort in python? @thesmartdeveloperr

**Comment by thesmartdeveloperr** [comment]
yes @TishaJhabak1014 you can do that for sure, we already have merge sort in C i think, just add your .py file in that folder and do a PR, would love to see it.

**Comment by TishaJhabak1014** [comment]
> yes @TishaJhabak1014 you can do that for sure, we already have merge sort in C i think, just add your .py file in that folder and do a PR, would love to see it.

Sure, also are we having hactoberfest labels here? : )

**Comment by Rohanfizz** [comment]
@TishaJhabak1014 I was planning to contribute Tortoise and Hare algorithm for linkedlists next thats why provided implementations.

**Comment by thesmartdeveloperr** [comment]
> > yes @TishaJhabak1014 you can do that for sure, we already have merge sort in C i think, just add your .py file in that folder and do a PR, would love to see it.
> 
> Sure, also are we having hactoberfest labels here? : )

yaa when i will merge your PR it will count in hacktoberfest...

**Comment by TishaJhabak1014** [comment]
> > > yes @TishaJhabak1014 you can do that for sure, we already have merge sort in C i think, just add your .py file in that folder and do a PR, would love to see it.
> > 
> > 
> > Sure, also are we having hactoberfest labels here? : )
> 
> yaa when i will merge your PR it will count in hacktoberfest...

@thesmartdeveloperr Awesome! I will head over to add an issue. Do add the labels : )
